internal/resources/dataprotection/converter_mapping.go

```diff
@@ -0,0 +1,77 @@
+/*
```

Comment:
Actually it's difficult to see in the github PR since it shows as file deleted + added instead of modified but there are some changes to add the cluster group scope support in these files.